packages/wallets/nufi/src/publicTypes.ts

```diff
@@ -0,0 +1,80 @@
+/* This file should be kept in sync between the NuFi and the `wallet-adapter`
+ * codebases. */
```

Comment:
Please inline this with the adapter file, and only include the types needed by the adapter. Docs aren't needed because it's an internal type.

Reply:
Thanks, I inlined it.